CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ All notable changes to this project will be documented in this file based on the
 ### Backward Compatibility Breaks
 
 ### Bugfixes
+- Characters "<" and ">" will be removed when a query term is passed to [`Util::escapeTerm`](https://github.com/ruflin/Elastica/pull/1415/files). Since v5.1 the [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/5.1/query-dsl-query-string-query.html#_reserved_characters) states that these symbols cannot be escaped ever.
+- Remove [`each()`](http://www.php.net/each) usage to fix PHP 7.2 compatibility
+
+### Added
+
+* Added request parameters to `Client->deleteDocuments()`. [#1419](https://github.com/ruflin/Elastica/pull/1419)
```

Comment:
I think a few more things sneaked into the changelog here?

Reply:
oops, sorry, I cherry-picked the commit and I didn't notice this 😬